share/completions/black.fish

```diff
@@ -0,0 +1,23 @@
+complete -c black -s c -l code -d 'Code to format'
+complete -c black -s l -l line-length -d 'Characters per line'
+complete -c black -s t -l target-version -d 'Python versions supported by output'
+complete -c black -l pyi -d 'Format all input files like typing stubs regardless of file extension'
+complete -c black -s S -l skip-string-normalization -d 'Don\'t normalize string quotes or prefixes'
+complete -c black -s C -l skip-magic-trailing-comma -d 'Don\'t use trailing commas as a reason to split lines'
+complete -c black -l check -d 'Don\'t write the files back, just return the status'
+complete -c black -l diff -d 'Don\'t write the files back, just output a diff for each file'
+complete -c black -l color -d 'Show colored diff'
+complete -c black -l no-color -d 'Do not show colored diff'
+complete -c black -l fast -d 'Skip temporary sanity checks'
+complete -c black -l safe -d 'Do not skip temporary sanity checks'
+complete -c black -l required-version -d 'Require a specific version of Black to be running'
+complete -c black -l include -d 'Regular expression of items to include'
+complete -c black -l exclude -d 'Regular expression of items to exclude'
+complete -c black -l extend-exclude -d 'Additional items to exclude'
+complete -c black -l force-exclude -d 'Items to always exclude'
+complete -c black -l stdin-filename -d 'The name of the file when passing through stdin'
+complete -c black -s q -l quiet -d 'Don\'t emit non-error messages to stderr'
```

Comment:
This should probably be changed upstream but wouldn't `Only print error messages to stderr` sound better?